### PR TITLE
Feature/segun/order details

### PR DESCRIPTION
**Pull request by Segun-Ade**
#### What does this PR achieve?
Order details
#### Screenshots:
DeepinScreenshot_select-area_20200121072826
DeepinScreenshot_select-area_20200121072750

### CODE REVIEW

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <div key={item.key}>
+            {/* eslint-disable-next-line react/jsx-no-literals */}
+            {`${GARMENT} ${item.garmentDesc}|
+            ${item.garment}`}
+            <br />
+            {`${PRICE} ${item.garmentPrice}`}
+            <br />
+            {`${QUANTITY} ${item.quantity}`}
+            <br />
+            {`${EST_TIME} ${item.estimatedTime}`}
```
Comment (ChineduOguejiofor) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: I think you can use string concatenation here 

```suggestion
          
   {EST_TIME + item.estimatedTime}
```

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                <span>
+                    {STATUS}
+                    <Tag color={color} href="#">
+                        {singleOrder.orderStatus}
+                    </Tag>
+                </span>
+            )}
+            // eslint-disable-next-line react/jsx-no-literals
+            title={`${VENDOR}${userInfo.displayName}`}
+        >
```
Comment (ChineduOguejiofor) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: Same here

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            ) : (
+                <Card>{NO_DETAILS}</Card>
+            )}
+        </Col>
+    );
+}
+const PropTable = singleOrder => {
+    const { userInfo } = singleOrder;
+    let color;
+    if (singleOrder.orderStatus === 'pending' || singleOrder.orderStatus === 'waiting') {
```
Comment (justiceotuya) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: distructure

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    const data = useSelector(state => state);
+    const {
+        orders: { singleOrder },
+    } = data;
+    return (
+        <>
+            <Row type="flex" justify="center" align="middle" gutter={[32, 32]}>
+                {PropTable(singleOrder)}
+            </Row>
+        </>
```
Comment (Way2nnadi) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: no need for this fragment here

File: src/components/userDashboard/components/orders/components/orderDetails.jsx (hunk 3 above)
Comment (Way2nnadi) [reply src/components/userDashboard/components/orders/components/orderDetails.jsx]: destructure `orderStatus` from `singleOrder`

File: src/components/userDashboard/components/orders/components/orderDetails.jsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        <div key={item.key}>
+            {GARMENT + item.garmentDesc}
+            <br />
+            {PRICE + item.garmentPrice}
+            <br />
+            {QUANTITY + item.quantity}
+            <br />
+            {EST_TIME + item.estimatedTime}
+            <br />
+            <br />
```
Comment (Way2nnadi) [inline src/components/userDashboard/components/orders/components/orderDetails.jsx]: remove all of these `br` and use css for your styles

**Pull request merged**